resources/lang/fr/pireps.php

```diff
@@ -1,12 +1,12 @@
 <?php
 
 return [
-    'newflightreport'      => 'Nouveau Rapport de vol',
-    'editflightreport'     => 'Editer le Rapport de vol',
+    'newflightreport'      => 'Nouveau Rapport de Pilote (PIREP)',
+    'editflightreport'     => 'Edition du Rapport de Pilote (PIREP)',
```

Comment:
Hm, "Rapport de vol" is a report about a flight while "Rapports de pilote" is a report about a pilot. I don't know what you think but "Rapports de vol" may be better since a PIREP is more about a flight flown by someone than someone.

Reply:
C'est juste pour se rapprocher de Pirep, qui signifie PIlots REPort

Reply:
Okay why not